src/status_im/contexts/wallet/send/select_address/view.cljs

```diff
@@ -157,7 +155,6 @@
                                         :on-press            #(rf/dispatch
                                                                [:wallet/select-send-address
                                                                 {:address @input-value
-                                                                 :token? (some? token)
```

Comment:
still not clear of why you are removing this?
we need this to know whether we are sending a collectible or a token 
cc @briansztamfater